src/main/java/appeng/me/cache/CraftingGridCache.java

```diff
@@ -489,7 +489,8 @@ public IAEItemStack extractItems( final IAEItemStack request, final Actionable m
 					if( ais.getItem() == whatToCraft.getItem() && ( !ais.getItem().getHasSubtypes() || ais.getItemDamage() == whatToCraft.getItemDamage() ) )
 					{
 						// TODO: check if OK
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)